README.md

```diff
@@ -6,106 +8,64 @@
 
 1. What is the first step for installing Dart on a Windows machine?
 
-A) Install Homebrew
-B) Download the Dart SDK
-C) Update your PATH
-D) Run Dart Doctor
-
+'C) Update your PATH '
 
 2. Which command verifies the Dart installation on macOS?
 
-A) dart --install
-B) dart --check
-C) dart --verify
 D) dart --version
 
 
 3. What is the next step after downloading and extracting the Flutter SDK on Linux?
 
-A) Install Homebrew
-B) Update your PATH
-C) Run Flutter Doctor
-D) Create a new Flutter project
-
+'C) Update your PATH '
 
 4. What command is used to run a newly created Flutter app?
 
-A) flutter start
-B) flutter build
-C) flutter run
-D) flutter init
+'C) flutter run '
 
 
 #Python Installation
 
 What is the first step to install Python on a Windows system?
 
-A) Run the installer without any customization
-B) Download Python from the official website
-C) Open the terminal and type sudo apt install python
-D) Install pip manually
+'B) Download Python from the official website '
 
 Which option should you ensure is checked during Python installation on Windows?
 
-A) Install with default settings
-B) Install to a custom directory
-C) Add Python to PATH
-D) Install all available features
+ 'C) Add Python to PATH '
+
 
 How do you verify Python installation on any system?
 
-A) By running python --version
-B) By restarting your computer
-C) By opening the Python installer again
-D) By checking the Programs and Features in Control Panel
+'A) By running python --version '
 
 What command is used to install pip on macOS and Linux?
 
-A) sudo install pip
-B) pip install python
-C) curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py
-D) python --install pip
+'C) curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py '
 
 What is the purpose of a virtual environment in Python?
```

Comment:
This is the first step to install python.

Reply:
B) Download Python from the official website

Reply:
B downoload python from official website

Reply:
B. download python from official website

Reply:
B. Download python from the offical website

Reply:
Download Python from the official website

Reply:
B downoload python from official website